fava/templates/_layout.html

```diff
@@ -101,6 +95,16 @@ <h1 class="site-name">{{ api.title or 'fava' }}{{ ' ▾' if config['APIS']|lengt
     </article>
     {% include "overlays/_keyboard_shortcuts.html" %}
     {% include "overlays/_documents_upload.html" %}
+    {% if config['APIS']|length > 1 %}
```

Comment:
I don't think the header should be fixed on mobile (it takes up much more space relatively). As a side effect, the stacking context wouldn't be a problem then. Or else the whole header can just be z-indexed above the sidebar, in which case this also isn't a problem. Then we can go back to the CSS dropdown as well.